src/cli/bin.ts

```diff
@@ -175,6 +286,21 @@ This will output a file named \`report.json\` containing the error messages.`);
       process.exit(0);
     },
   )
+  .check(({ fromVersion, toVersion }) => {
+    const fromVersionIndex = migrationFunctions.findIndex(
+      ([version]) => version === fromVersion,
+    );
+    const toVersionIndex = migrationFunctions.findIndex(
+      ([version]) => version === toVersion,
+    );
+    if (fromVersionIndex >= toVersionIndex) {
```

Comment:
If possible let's just use `semver`:
```suggestion
    if (semver.gte(fromVersion, toVersion)) {
```